config/initializers/launch_darkly.rb

```diff
@@ -0,0 +1 @@
+$ld_client = LaunchDarkly::LDClient.new(ENV.fetch("LAUNCHDARKLY_SDK_KEY"))
```

Comment:
I know the `$ld_client` global variable name is what their docs suggest, but I'm thinking it might be easier for future devs to pick up what's going on if we rename this to something like `$launch_darkly` instead?

Reply:
This is still missing the `$launch_darkly` definition?